Unable to install `gogen-avro` since `v5.1.0`

It seems that import paths are broken in the new tag:

```
go get gopkg.in/actgardner/gogen-avro.v5
go install gopkg.in/actgardner/gogen-avro.v5/gogen-avro
/go/src/gopkg.in/actgardner/gogen-avro.v5/gogen-avro/main.go:11:2: cannot find package "github.com/actgardner/gogen-avro/generator" in any of:
        /usr/local/go/src/github.com/actgardner/gogen-avro/generator (from $GOROOT)
        /go/src/github.com/actgardner/gogen-avro/generator (from $GOPATH)
/go/src/gopkg.in/actgardner/gogen-avro.v5/gogen-avro/main.go:12:2: cannot find package "github.com/actgardner/gogen-avro/types" in any of:
        /usr/local/go/src/github.com/actgardner/gogen-avro/types (from $GOROOT)
        /go/src/github.com/actgardner/gogen-avro/types (from $GOPATH)
```

I see the following change between `5.0.1` and `5.1.0`:

https://github.com/actgardner/gogen-avro/blob/v5.0.1/gogen-avro/main.go#L11
https://github.com/actgardner/gogen-avro/blob/v5.1.0/gogen-avro/main.go#L11

Any chance a new tag would fix this?

Thanks!

--- a/release.sh
+++ b/release.sh
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-# Rewrite references from github.com/alanctgardner/gogen-avro to gopkg.in/alanctgardner/gogen-avro.<version>
+# Rewrite references from github.com/actgardner/gogen-avro to gopkg.in/actgardner/gogen-avro.<version>
 
 if [ "$#" -ne 1 ]; then
   echo "Usage: $0 <version>"
   exit 1
 fi
  
-GITHUB_REPO="github.com/alanctgardner/gogen-avro"
+GITHUB_REPO="github.com/actgardner/gogen-avro"
 VERSION="$1"
-GOPKG_REPO="gopkg.in/alanctgardner/gogen-avro.$VERSION"
+GOPKG_REPO="gopkg.in/actgardner/gogen-avro.$VERSION"
 
 sed -i "s|$GITHUB_REPO|$GOPKG_REPO|" container/*.go generator/*.go types/*.go gogen-avro/main.go example/*/*.go test.sh 

